@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -1616,7 +1597,8 @@ export class TextElementsRenderer {
             this.m_viewState.lookAtDistance
         );
         const iconReady =
-            renderIcon && poiRenderer?.prepareRender(pointLabel, this.m_viewState.env) === true;
+            renderIcon &&
+            this.m_poiRenderer.prepareRender(pointLabel, this.m_viewState.env) === true;
```

Comment:
no need for the trailing "=== true", since no poiRenderer is always defined.

Reply:
done